spec/awful/completion_spec.lua

```diff
@@ -5,18 +5,29 @@ local shell = function(...)
 end
 local gfs = require("gears.filesystem")
 
+local test_dir
+
+--- Get and create a temporary test dir based on `pat`, where %d gets replaced by
+-- the current PID.
+local function get_test_dir(pat)
+    local pid = io.open("/proc/self/stat", "r"):read():gmatch("%S+")()
```

Comment:
Yup, looks good. However, your `fixup!` is for the wrong commit, I think. Besides that, 👍 .